source/Cosmos.HAL2/Drivers/PCI/Audio/AudioManager.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Cosmos.HAL.Drivers;
+using Cosmos.HAL.Drivers.PCI.Audio.Generic;
+using System;
+using System.Collections.Generic;
+
+namespace Cosmos.HAL.Drivers.PCI.Audio
+{
+    /// <summary>
+    /// Audio mananger class. Right now, it only supports sound blaster 16.
+    /// </summary>
+    public class AudioManager
+    {
+        public static List<SoundCard> devices = new List<SoundCard>();
+
+        public static void Init()
+        {
+            devices.Clear(); //just in case
+            try
+            {
+                Console.WriteLine("Searching for sound cards");
```

Comment:
Cosmos itself shouldnt be writing to the console, since that is something the individual os should be able to control.